jib-core/src/main/java/com/google/cloud/tools/jib/image/LayerEntry.java

```diff
@@ -61,10 +79,11 @@ public LayerEntry(Path sourceFile, Path extractionPath) {
   /**
    * Gets the source file.
    *
-   * <p>Do <b>not</b> call {@link Path#toString} on this - use {@link #getSourceFileString} instead.
-   * This path can be relative or absolute, and {@link #getSourceFileString} can also be relative or
-   * absolute, but callers should rely on {@link #getSourceFileString} for the serialized form since
-   * the serialization could change independently of the path representation.
+   * <p>Do <b>not</b> call {@link Path#toString} on this - use {@link #getAbsoluteSourceFileString}
+   * instead. This path can be relative or absolute, and {@link #getAbsoluteSourceFileString} can
+   * also be relative or absolute, but callers should rely on {@link #getAbsoluteSourceFileString}
```

Comment:
"can also be relative or absolute"

It can no longer be relative.

Reply:
The path representation can still be relative or absolute, but the serialized string form is always absolute.

Reply:
Sorry, I was referring to

"and {@link #getAbsoluteSourceFileString} can also be relative or absolute"

I believe "getAbsoluteSourceFileString" is always absolute.

Reply:
Ah right, I'll fix that.